nym-api/src/nym_nodes/handlers/unstable/skimmed.rs

```diff
@@ -124,6 +124,21 @@ where
     // (ideally it'd be tied directly to the NI iterator, but I couldn't defeat the compiler)
     let describe_cache = state.describe_nodes_cache_data().await?;
 
+    let maybe_interval = state
+        .nym_contract_cache()
+        .current_interval()
+        .await
+        .to_owned();
+
+    // 4.0 If the client indicates that they already know about the current topology send empty response
```

Comment:
just thinking aloud here: I guess in the longer term, there'd need to be some even further identifier to distinguish between updates mid-epoch, e.g. some node changed its ip address or port, etc. because ideally the client should receive those as soon as possible rather than having to wait until next epoch